src/pyfmi/fmi.pyx

```diff
@@ -693,8 +703,9 @@ cdef class ModelBase:
             if self._max_log_size_msg_sent:
                 return
 
-            msg = "The log file has reached its maximum size and further log messages will not be saved. To change the maximum size of the file, please use the 'set_max_log_size' method."
+            msg = "The log file has reached its maximum size and further log messages will not be saved. To change the maximum size of the file, please use the 'set_max_log_size' method.\n"
             self._max_log_size_msg_sent = True
+            self._current_log_size = self._current_log_size + len(msg)
```

Comment:
We _could_, I'd rather not though, it will make the code quite messy due to the various ways how one can log. I think in scenarios where writing to stream/file fails, it is acceptable to not be 100% accurate in hitting the limit.

Reply:
Sounds good to me!